crabs/detection_tracking/train_model.py

```diff
@@ -52,11 +53,19 @@ def train_model(self):
             self.main_dirs, annotations, self.config, self.seed_n
         )
 
-        lightning_model = FasterRCNN(self.config)
+        timestamp = datetime.datetime.now().strftime("%Y-%m-%d_%H-%M-%S")
+        run_name = f"experiment_{timestamp}"
```

Comment:
```suggestion
        run_name = f"run_{timestamp}"
```
I find it confusing to call the run `experiment_`, since in MLflow an experiment has multiple runs. Maybe simply `run_`?